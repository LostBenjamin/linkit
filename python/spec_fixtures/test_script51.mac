# ------- test 1 -----------
%include test_script51_include

[testmacro]
  blue

  (testmacro)
  (test_script51_include-test1macro)
  (test_script51_include2-test2macro)

