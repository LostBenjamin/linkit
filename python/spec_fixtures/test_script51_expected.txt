4:set:blu:4:run:6:run:8:run
5:set:red
6:set:grn:6:run:8:run
7:set:pur
8:set:red:8:run

